src/backend/tcop/postgres.c

```diff
@@ -1645,7 +1645,7 @@ exec_simple_query(const char *query_string, const char *seqServerHost, int seqSe
 		querytree_list = pg_analyze_and_rewrite(parsetree, query_string,
 												NULL, 0);
 
-		plantree_list = pg_plan_queries(querytree_list, NULL, false, QRL_ONCE);
+		plantree_list = pg_plan_queries(querytree_list, NULL, true, QRL_ONCE);
 
```

Comment:
It's not right
```
A snapshot must be set before invoking the planner, since it
 * might try to evaluate user-defined functions.  But we must not set a
 * snapshot if the list contains only utility statements, because some
 * utility statements depend on not having frozen the snapshot yet.
```

Snapshot for plannable statement was set before, see line 1631.